apps/etterlatte-brev-api/.nais/azuread-etterlatte-brev-api-lokal.yaml

```diff
@@ -0,0 +1,29 @@
+######################
```

Comment:
Må vi ha en slik pr app for å få det til å snurre lokalt?

Reply:
Du kan droppe noe av det hvis appen _kun_ kjører lokalt, men letteste er å ha det med siden man bare kjører scriptet. 😄 

Slo meg også at jeg helt glemte å nevne at folk burde installere plugin [EnvFile](https://plugins.jetbrains.com/plugin/7861-envfile) for å støtte .env-filer i backendkoden. Skal oppdatere README!

Reply:
👍